certified-connectors/sendSMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,248 @@
+{
+  "swagger": "2.0",
```

Comment:
Kindly update below two properties. 

The 'contact' property is required. ValuePath: /info
The 'x-ms-connector-metadata' property is required.

Please refer the template from the link below. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json